app/assets/javascripts/katello/hosts/activation_key_edit.js

```diff
@@ -127,51 +120,24 @@ function ktAkTab() {
 }
 
 function ktOnLoad() {
+    tfm.store.observeStore('typeAheadSelect', (items) => {
+        if (items.kt_activation_keys) {
+            var selected = items.kt_activation_keys.selected;
+            selected = selected ? selected : [];
+
+            ktAkUpdateSubscriptionsInfo(selected);
+            ktSetParam(KT.KT_AK_LABEL, selected.join(','));
+        }
+    });
+
     ktHideParams();
     ktParamToAkInput();
     ktLoadActivationKeys();
 }
 
 $(document).on('ContentLoad', function(){
-
-    $("#kt_activation_keys").parents("form").submit(ktAkInputToParam);
```

Comment:
the host/hostgroup parameter is now updated by listening for changes in the redux store rather than on submit